blog/service/core/src/main/kotlin/com/excref/kotblog/blog/service/post/PostService.kt

```diff
@@ -0,0 +1,36 @@
+package com.excref.kotblog.blog.service.post
+
+import com.excref.kotblog.blog.service.post.domain.Post
+
+/**
+ * @author Rafayel Mirzoyan
+ * @since 6/10/17 6:58 PM
+ */
+interface PostService {
+    /**
+     * Creates a new post with the given ...
+     * @param   name The post name
+     * @param   title The post title
+     * @param   content The post content
+     * @param   blogUuid The post blog uuid
+     * @param   tagUuids The post tags uuids
+     * @param   categoryUuids The post categories uuids
+     * @return  Post if created successfully
+     */
+    fun create(name: String,
```

Comment:
I would suggest to wrap this arguments in to `CreatePostDto` or `PostCreationDto`
TODO: let's discuss